scripts/run_diff.sh

```diff
@@ -37,7 +37,7 @@ then
     echo "Ensure cpplint.py is inside the $script_folder directory then run again"
     exit 1
   else
-    cmd='${script_folder}/cpplint.py $file 2>&1 >/dev/null'
+    cmd='${script_folder}/cpplint.py $file --filter=-whitespace,-readability/identifier_spacing 2>&1 >/dev/null'
```

Comment:
Does this disable all whitespace checks? Didn't you want to disable only those for assignment operators?

Reply:
Fixed.